pyproject.toml

```
@@ -27,7 +27,7 @@ exclude = [".git", "__pycache__", ".tox", "build", "dist", "*.egg_info", "venv"]
 select = ["E", "W", "F", "C", "N", "R", "D", "H"]
 # Ignore W503, E501 because using black creates errors with this
 # Ignore D107 Missing docstring in __init__
-ignore = ["W503", "E501", "D107"]
+ignore = ["W503", "E501", "D107", "N806"]
```

Yay, this can be dropped -- see below

```suggestion
ignore = ["W503", "E501", "D107"]
```